### PR TITLE
docs: fix an error in the README

### PR DESCRIPTION
**Pull request by jajoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ var Xtest = new Matrix([[0, -2], [1, 0.5], [1.5, -1], [1, 2.5], [2, 3.5], [1.5,
 var Ytest = Matrix.columnVector([0, 0, 0, 1, 1, 1, 2, 2, 2]);
 
 // we will train our model
-var logreg = new LogisticRegression(numSteps = 1000, learningRate = 5e-3);
+var logreg = new LogisticRegression({numSteps = 1000, learningRate = 5e-3});
```
Comment (stropitek) [inline README.md]: This is still invalid syntax...

Comment (jajoe) [inline README.md]: But it works... you prefer to do a variable options ?

Comment (maasencioh) [inline README.md]: `var logreg = new LogisticRegression({numSteps: 1000, learningRate: 5e-3});`

**Comment by jajoe** [comment]
You were right... error of copy/paste. I've changed that.

**Pull request merged**